MagazineLayout/LayoutCore/ModelState.swift

```diff
@@ -611,6 +664,47 @@ final class ModelState {
     return .updateCurrentModels
   }
 
+  private func preferredHeightUpdateContextForPreferredHeightUpdateToSupplementaryView(
```

Comment:
Could simplify this to something like `updateContextForSupplementaryViewPreferredHeightUpdate`